### ISSUE TITLE
The example is in German

### ISSUE THREAD

**Issue by Haroenv** [FauDroids/MyAdvent#30]
I don't think that's a really big deal, but for the sake of consistency, wouldn't it be better if the demo was in English?

**Comment by ohaz** [comment]
You're right. We should have two examples, one in German and one in English, depending on the language of the browser of the user.
